app/full_text_search.py

```diff
@@ -42,13 +56,21 @@ def search(repo, query_txt, max_results):
     results = []
     if not query_txt:
         return results
-    index = appengine_search.Index(name=PERSON_FULL_TEXT_INDEX_NAME)
-    query_txt += ' AND (repo: ' + repo + ')'
+    person_index = appengine_search.Index(name=PERSON_FULL_TEXT_INDEX_NAME)
+    person_location_index = appengine_search.Index(
+        name=PERSON_LOCATION_FULL_TEXT_INDEX_NAME)
     options = appengine_search.QueryOptions(
         limit=max_results,
         returned_fields=['record_id'])
-    index_results = index.search(appengine_search.Query(
-        query_string=query_txt, options=options))
+    or_query = create_or_query(query_txt) + ' AND (repo: ' + repo + ')'
+    person_index_results = person_index.search(appengine_search.Query(
+        query_string=or_query, options=options))
+    query_txt += ' AND (repo: ' + repo + ')'
```

Comment:
Modifying a parameter can be a cause of bugs. (e.g., If you happen to put this line before line 65, it breaks.) Can you define a new variable instead of modifying query_txt?